owl/EASE-ACT.owl

```diff
@@ -466,6 +562,32 @@ In this category we will place Actions for which such extra knowledge is require
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#ManipulationTask -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#ManipulationTask">
```

Comment:
This is a tricky one :P To follow the convention of most Task names, it should be Manipulating. The problem is that word carries a very strong connotation, of persuading someone to do something through perhaps not legitimate means.

Reply:
yes that's true. But by appending *Task* this ambiguity is not solved!

Reply:
What would be a good name then? In any case it needs to contain manipulation in some way.

Reply:
*manual actuation* seems to exist, as in you actuate by using your own hands.
But manual manipulation sounds silly, hand manipulation as well.

Reply:
ok it's a hard one. Manipulation comes from early 18th century from French, from Latin manipulus ‘handful’. So it inherently refers to 'hand'. So I would argue that any other use of the word, e.g. as in manipulating the opinion of someone is rather metaphorically because you do not use your hands

Reply:
Ok, then I'll just use Manipulating.

Reply:
yes, maybe add a comment about the word origin manipulation--manual--using the hands

Reply:
Done.